### DISCUSSION TITLE
Home Assistant API failing "Missing price data"

### DISCUSSION THREAD

**Discussion by chiefcomm** [amberelectric/public-api#209] [Home Assistant]
Hi Myles,

I noticed this morning (30th June) the Home Assistant call is failing since 12:00am (almost to the second).

Below error in logs.

````
2024-06-30 10:51:47.706 DEBUG (SyncWorker_62) [amberelectric.rest] response body: b'{"message": "Internal server error"}'
2024-06-30 10:51:47.708 DEBUG (MainThread) [homeassistant.components.amberelectric] Finished fetching amberelectric data in 0.973 seconds (success: False)
2024-06-30 10:51:47.708 DEBUG (MainThread) [homeassistant.config_entries] Config entry '2115' for amberelectric integration not ready yet: Missing price data, skipping update; Retrying in 20 seconds
````

**Comment by madpilot** [comment]
We are having issues pricing a couple of tariffs at the moment. Well applying the fixes tomorrow.

**Comment by chiefcomm** [comment]
Thanks Myles, no exports on a rainy day like today so not a big deal :-)

**Comment by l33cuj3s** [comment]
Ah, glad someone pointed me to this. Had commenced troubleshooting and posting [results](https://www.facebook.com/groups/318367954142793/permalink/461784113134509/) on FB.

**Comment by chiefcomm** [comment]
Hi Myles,
Any update on when this will be resolved?
Thanks

**Comment by madpilot** [comment]
Still working on it.

**Reply by madpilot** [reply]
What tariff are you on? I can give. You a better idea of where in the queue that is

**Reply by l33cuj3s** [reply]
For myself, it's as follows:
![Screen Shot 2024-06-30 at 11 08 51 am](https://github.com/amberelectric/public-api/assets/145726411/6aae70c7-788f-43b2-9ac9-fe84dceaf427)

**Reply by madpilot** [reply]
N73 is the big one we are still working on.

**Reply by chiefcomm** [reply]
Sadly also N73 Endeavour Energy :-)

**Reply by l33cuj3s** [reply]
Looks like we're back in action. Bravo Myles!

**Reply by madpilot** [reply]
How are you faster than my CI?

**Reply by chiefcomm** [reply]
Fantastic news thanks Myles

**Reply by l33cuj3s** [reply]
@madpilot Just lucky I think! Happened to glance at my dashboard which was suddenly populated.
